hoshino/modules/weibo/weibo.py

```diff
@@ -0,0 +1,456 @@
+        logger.info(u'粉丝数：%d' % self.user['followers_count'])
+        logger.info(u'关注数：%d' % self.user['follow_count'])
+        logger.info(u'url：https://m.weibo.cn/profile/%s' % self.user['id'])
+        if self.user.get('verified_reason'):
+            logger.info(self.user['verified_reason'])
+        logger.info(self.user['description'])
+        logger.info('+' * 100)
+
+    def get_one_weibo(self, info):
+        """获取一条微博的全部信息"""
+        try:
+            weibo_info = info['mblog']
+            weibo_id = weibo_info['id']
+            retweeted_status = weibo_info.get('retweeted_status')
+            is_long = weibo_info.get('isLongText')
+            if retweeted_status and retweeted_status.get('id'):  # 转发
+                retweet_id = retweeted_status.get('id')
+                is_long_retweet = retweeted_status.get('isLongText')
+                if is_long:
+                    weibo = self.get_long_weibo(weibo_id)
+                    if not weibo:
+                        weibo = self.parse_weibo(weibo_info)
+                else:
+                    weibo = self.parse_weibo(weibo_info)
+                if is_long_retweet:
+                    retweet = self.get_long_weibo(retweet_id)
+                    if not retweet:
+                        retweet = self.parse_weibo(retweeted_status)
+                else:
+                    retweet = self.parse_weibo(retweeted_status)
+                retweet['created_at'] = self.standardize_date(
+                    retweeted_status['created_at'])
+                weibo['retweet'] = retweet
+            else:  # 原创
+                if is_long:
+                    weibo = self.get_long_weibo(weibo_id)
+                    if not weibo:
+                        weibo = self.parse_weibo(weibo_info)
+                else:
+                    weibo = self.parse_weibo(weibo_info)
+            weibo['created_at'] = self.standardize_date(
+                weibo_info['created_at'])
+            return weibo
+        except Exception as e:
+            logger.exception(e)
+            self.__recent = False
+
+    def get_latest_weibos(self):
+        try:
+            latest_weibos = []
+            js = self.get_weibo_json(1)
+            if js['ok']:
+                weibos = js['data']['cards']
+                for w in weibos:
+                    if w['card_type'] == 9:
+                        wb = self.get_one_weibo(w)
+                        if wb:
+                            if not self.__recent:
+                                continue
+                            if (not self.filter) or (
+                                    'retweet' not in wb.keys()):
+                                latest_weibos.append(wb)
+                                self.print_weibo(wb)
+                            
+            return latest_weibos
+        except Exception as e:
+            logger.exception(e)
+            return []
+
+
+user_configs = util.load_config(__file__)
```

Comment:
A sample config file is requested.

Reply:
A short sample file is in `weibo.py` now